source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,40 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
+## Install
```

Comment:
Add an empty line above this and use the `{% linkable_title %}` tag.
I would suggest `###` instead of `##` to be in line with other pages.

Reply:
done